inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/queue/StreamQueueResourceListener.java

```diff
@@ -83,6 +85,9 @@ public ListenerResult listen(WorkflowContext context) throws WorkflowListenerExc
             log.error(msg);
             throw new WorkflowListenerException(msg);
         }
+        GroupStatus groupStatus = GroupStatus.forCode(groupInfo.getStatus());
+        Preconditions.expectTrue(GroupStatus.CONFIG_FAILED != groupStatus,
```

Comment:
An error code needs to be carried when an exception is thrown

Reply:
This doesn't need to be set, it doesn't get thrown out.